content/news/buzz/come-sprint-with-us-at-pycon-us-2017/contents+es.lr

```diff
@@ -26,7 +26,7 @@ Con este stand, tenemos dos entradas para la conferencia. Esto incluye:
 - 3 días de conferencias, salón de exposiciones/feria de trabajo
 - desayunos, pausas, almuerzos y bolsa de *swag*
 
-La cosa es que tanto `Russell <https://twitter.com/freakboy3742>`__ como yo, ya nos hemos registrado.
+La cosa es que tanto `Russell <https://mastodon.social/@freakboy3742@cloudisland.nz>`__ como yo, ya nos hemos registrado.
```

Comment:
This is a historical blog post; Updating the author link is fine; but we shouldn't be retrospectively editing the blog post content.